packages/discovery-react-components/src/components/SearchFacets/utils/searchFilterTransform.tsx

```diff
@@ -83,7 +83,7 @@ export class SearchFilterTransform {
   private static unquoteString(quotedString: string): string {
     return quotedString
       .replace(new RegExp(SearchFilterTransform.STRING_IN_QUOTES), '$1')
-      .replace(/\\"/, '"');
+      .replace(/\\"/g, '"');
```

Comment:
What's up with these changes? They make sense, just wondering how we didn't catch them before.

Reply:
Oops, adding this change to this PR was actually accidental-- I had made it when investigating the issue Fucheng had pointed out last week. We can leave it here or wait until we're fixing that issue. (This change alone doesn't fix it.)

Reply:
If not specifically for this (and you'll handle it later), I would prefer to remove it.